### ISSUE TITLE
Tip/request: send 2captcha urls with the requests to 2captcha 

### ISSUE THREAD

**Issue by ultrafunkamsterdam** [novskey/PalletTown#19]
### important: This is not an Issue, but information to -or not- anticipate on ###
Hi There,
Great application you wrote. Finally automated acc creation. Props for that. Some info where you could/ -or not- not anticipate to:

Here's the printscreen:
![image](https://cloud.githubusercontent.com/assets/21027969/23924418/eac73e54-090a-11e7-80b7-22c226e0935c.png)

I get several of these emails when using PT, but not in other ... ugghgrocketmapsugggg.. scripts. 
Do what you want with it and keep up the good work 👍 👍 👍

**Comment by Jeroen196** [comment]
Hey @ultrafunkamsterdam ! 
Same problem here, getting those mails. However for me this is a problem, since trying to use autosolving with 2Captcha no longer succesfully creates accounts. When I create accouts using manual verification they all succeed, but when using a 2Captcha key it does validate the captcha (after 3 tries), but fails to create the account. 

Are you able to create accounts and verify them using 2Captcha?

-- Debug info --

> [19:37:54] Worker 3 did 1 tasks
[19:38:19]     [DEBUG]:[Worker 2]:  initializing..
[19:38:19]     [DEBUG]:[Worker 2]: Attempting to create user <username>:<password>. Opening browser...
[19:38:19]     [DEBUG]:[Worker 2]: Proxy type: 
[19:38:19]     [DEBUG]:[Worker 2]: 2captcha key
[19:38:19]     [DEBUG]:None
[19:38:19]     [DEBUG]:Step 1: Verifying age using birthday: 1975-05-22
[19:38:19]     [DEBUG]:[Worker 2]: trying to execute workaround script
[19:38:19]     [DEBUG]:[Worker 2]: done executing workaround script, submitting dob
[19:38:19]     [DEBUG]:[Worker 2]: dob submitted
[19:38:19]     [DEBUG]:[Worker 2]: Step 2: Entering account details
[19:38:19]     [DEBUG]:[Worker 2]: https://club.pokemon.com/us/pokemon-trainer-club/parents/sign-up
[19:38:19]     [DEBUG]:[Worker 2]: https://club.pokemon.com/us/pokemon-trainer-club/parents/sign-up
[19:38:19]     [DEBUG]:[Worker 2]: Starting autosolve recaptcha
[19:38:19]     [DEBUG]:[Worker 2]: We will wait 10 seconds for captcha to be solved by 2captcha
[19:38:19]     [DEBUG]:[Worker 2]: Captcha still not solved, waiting another 10 seconds.
[19:38:19]     [DEBUG]:[Worker 2]: Captcha still not solved, waiting another 10 seconds.
[19:38:19]     [DEBUG]:[Worker 2]: Captcha still not solved, waiting another 10 seconds.
[19:38:19]     [DEBUG]:[Worker 2]: Captcha still not solved, waiting another 10 seconds.
[19:38:19]     [DEBUG]:[Worker 2]: Captcha still not solved, waiting another 10 seconds.
[19:38:19]     [DEBUG]:[Worker 2]: Solved captcha
[19:38:19]     [DEBUG]:[Worker 2]: trying to submit
[19:38:19]     [DEBUG]:[Worker 2]: submitted
[19:38:19]     [DEBUG]:[Worker 2]: trying to validate response
[19:38:19]     [DEBUG]:[RESPONSE_VALIDATOR]: https://club.pokemon.com/us/pokemon-trainer-club/parents/sign-up
[19:38:19]     [DEBUG]:[RESPONSE_VALIDATOR]: Username already in use
[19:38:19]     [DEBUG]:[Worker 2]: Failed to create user: <username>...
[19:38:19]     [DEBUG]:[Worker 2]: threw failed to create user exception, terminate
[19:38:19] [Worker 2]: threw failed to create user exception, terminate
[19:38:19] Account 1 failed

>

**Comment by ultrafunkamsterdam** [comment]
Verify refers to clicking the link in the email. Has nothing to do with 2captcha. I don't use autoverify, mainly because i dont want to provide access to my mailbox (which you do by typing the mailbox's username and password to autoverify). Instead i forward all mails to my server. The simplest form just apppends all mails to a single file. I wrote a script to extract all links and activates them 1 by one

**Comment by parislarkins** [comment]
Yes tree is a problem with 2captcha creation in pallettown atm, however from limited testing if doesn't seem that the URL is the issue. When I get time I will be remaing PalletTown and these issues will be fixed, sorry about that

**Comment by Zyrix** [comment]
check out https://github.com/novskey/PalletTown/pull/25
